src/components/Tree/TreeItem/DragHandle.tsx

```diff
@@ -24,6 +24,7 @@ export const DragHandle = forwardRef(
                     active && 'tw-opacity-100 tw-text-white',
                     className,
                 ])}
+                data-test-id="tree-item-drag-handle"
```

Comment:
since we are adding the test-id now, it should be prefixed with `fondue-`

Reply:
done